packages/static-module-record/src/transform-analyze.js

```diff
@@ -122,25 +85,25 @@ const makeCreateStaticRecord = transformSource =>
     if (url) {
       functorSource += `//# sourceURL=${url}\n`;
     }
-
     const moduleAnalysis = freeze({
       exportAlls: freeze(sourceOptions.exportAlls),
       imports: freeze(sourceOptions.imports),
       liveExportMap: freeze(sourceOptions.liveExportMap),
       fixedExportMap: freeze(sourceOptions.fixedExportMap),
+      importMetaUttered: sourceOptions.importMeta.uttered,
```

Comment:
I believe we agree that we should pretend “utter” isn’t a word.

Reply:
That was 2 weeks after I put it here based on the sophisticated-vocabulary-filled meeting ;)
Then again - this is an internal name visible to people who need to grow their vocabulary significantly to grok this codebase anyway.  
I think the best result would be to make it consistent with the public name we eventually land.
https://github.com/tc39/proposal-compartments/pull/46